lib/transactions/crypto.js

```diff
@@ -102,8 +100,8 @@ function getBytes(transaction, skipSignature, skipSecondSignature) {
 		}
 	}
 	else if (transaction.vendorField) {
-		var vf = new Buffer(transaction.vendorField);
-		var fillstart=vf.length;
+		vf = new Buffer(transaction.vendorField);
```

Comment:
these variables need not be defined before the scope?

```
let vf;
if (true) vf = true;
else vf = false;
```

Reply:
`var` isn't blocked-scoped in JavaScript so I don't think they need to be defined ahead in the file. I moved the `var` defines ahead anyway to improve readability.